src/tools/rest-frontend/resources/assets/js/controllers/main/auth/AuthRegistrationController.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = function($scope, Logger, $state, $http, Notification, config) {
+
+	var vm = this;
+
+	$scope.user = {};
+
+
+	this.doRegistration = function() {
+		$http.post(config.backend.root + 'user', $scope.user, {
```

Comment:
I have to initialize the variable with an empty object as the corresponding HTML uses members of it, e.g. `user.email`. But don't worry, the frontend takes care of validation and does not allow an empty form to be submitted and therefore also no empty object gets sent. In case someone bypasses the validation, nothing happens as well - the server validates everything again anyway.